src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -23,6 +25,7 @@
     "phone" : "95352563",
     "email" : "heinz@example.com",
     "address" : "wall street",
+    "matriculationYear": "0000",
     "birthday" : "2000-03-03",
```

Comment:
Perhaps can swap the order of `matriculationYear` and `birthday` to ensure consistency.

Reply:
Thanks for the suggestion, you're right! I have updated the code accordingly with the suggested changes.